stages/05.adi-tools/13.install-linux_image_ADI-scripts/run.sh

```diff
@@ -11,5 +11,8 @@ chroot "${BUILD_DIR}" << EOF
 
 	# Clone linux_image_ADI-scripts
 	git clone -b kuiper2.0 ${GITHUB_ANALOG_DEVICES}/linux_image_ADI-scripts.git
+
+	# Install linux_image_ADI-scripts
+	cd linux_image_ADI-scripts && make -j $NUM_JOBS
```

Comment:
Wouldn't you need also the 'make install' command here?
Make install will copy the scripts from this repo in common paths so they can be run from any terminal location.

Reply:
The Makefile only has the "install" field, so by default make command takes the first (and only) field and makes it default, so "make" and "make install" are the same in this case.